blueskyacademy/bluesky-nextjs
# scroll into form and add 404 page

## Description
- Add form to register bus service and entrance exam
- Add 404 Page

## Why
- Add form to register bus service and entrance exam
- Show 404 Page when page is not found

## How to QA
- User can see 3 types of form in Parent Pages
- User can see 404 Page when page is not found



## Screenshots
![Screen Shot 2021-12-27 at 15 01 34](https://user-images.githubusercontent.com/74655110/147449691-d8e7e686-0763-4a2b-a1a0-36c05c47ab62.png)
![Screen Shot 2021-12-27 at 15 01 45](https://user-images.githubusercontent.com/74655110/147449701-0eddc03d-c5ae-4ac4-84f9-8ddc98e39b96.png)
